cmd/webhook/main.go

```diff
@@ -191,6 +192,28 @@ func newConversionController(ctx context.Context, cmw configmap.Watcher) *contro
 	)
 }
 
+func NewAuthorizationValidatingAdmissionController(ctx context.Context, cmw configmap.Watcher) *controller.Impl {
```

Comment:
this could be further broken out into a separate webhook command / deployment if that level of pluggability / opt in was desired